lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -6,11 +6,9 @@ module DateHelper
       alias_method :_distance_of_time_in_words, :distance_of_time_in_words
       alias_method :_time_ago_in_words, :time_ago_in_words
 
-      include DOTIW::Methods
-
       def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options = {}, options = {})
         return _distance_of_time_in_words(from_time, to_time, options) if options.delete(:vague)
-        super
+        DOTIW::Methods.distance_of_time_in_words(from_time, to_time, include_seconds_or_options, options)
```

Comment:
We're now able to call `DOTIW::Methods.distance_of_time_in_words` directly here thanks to Ruby's `module_function` method.

https://apidock.com/ruby/Module/module_function

Reply:
Let's use `extend self` instead. There's [a debate](https://github.com/rubocop-hq/ruby-style-guide/issues/556) on which one is better, but `extend self` always felt cleaner to me and in this case the different between `extend self` and `module_function` doesn't impact functionality.

Reply:
Done: https://github.com/radar/distance_of_time_in_words/pull/111/commits/be9ac5abe75b8a027648cab62fcf8affe82e0d55